lib/publiccloud/azure.pm

```diff
@@ -162,19 +201,56 @@ sub calc_img_version {
     return "$build.$kiwi";
 }
 
+sub generate_tags {
+    # Define tags
+    my $job_id = get_current_job_id();
+    my $openqa_url = get_required_var('OPENQA_URL');
+    my $created_by = "$openqa_url/t$job_id";
+    my $tags = "openqa_created_by=$created_by";
```

Comment:
```suggestion
    my $tags = "openqa_created_by=$created_by  openqa_var_server=$openqa_url openqa_var_job_id=$job_id";
```

Reply:
I just copied this part. Did you merge something in the mean time?

Reply:
> I just copied this part. Did you merge something in the mean time?

No.